SG14_test/inplace_function_test.cpp

```diff
@@ -429,6 +429,57 @@ static void RvalueRefParameter()
     g(std::make_unique<int>(42));
 }
 
+static void test_is_convertible()
+{
+    static_assert(std::is_convertible<int(&)(), stdext::inplace_function<int()>>::value, "");
+    static_assert(std::is_convertible<int(*)(), stdext::inplace_function<int()>>::value, "");
+    static_assert(std::is_convertible<int(*&)(), stdext::inplace_function<int()>>::value, "");
+    static_assert(std::is_convertible<int(*&&)(), stdext::inplace_function<int()>>::value, "");
+}
+
+namespace {
+struct InstrumentedCopyConstructor {
+    static int copies;
+    static int moves;
+    InstrumentedCopyConstructor() = default;
+    InstrumentedCopyConstructor(const InstrumentedCopyConstructor&) {
+        copies += 1;
+    }
+    InstrumentedCopyConstructor(InstrumentedCopyConstructor&&) {
+        moves += 1;
+    }
```

Comment:
This is a mix of 1TBS and Allman, let's pick one and stick to it.

Reply:
I'd say this style is just copied from `ThrowingFunctor` and so on from higher up in the file, and it seems consistent to me. If you want to clang-format the code later, I'm okay with that.